src/contexts/SwapFormContext/SwapFormContext.tsx

```diff
@@ -764,7 +764,7 @@ export const SwapFormProvider: React.FunctionComponent<SwapFormProviderProps> =
     // Check that the input margin is >= minimum required margin if removing margin
     if (
```

Comment:
this is duplicate... also found in the MintForm. Create a utility and share it across both components.